src/internal/Subscription.ts

```diff
@@ -144,10 +144,28 @@ export class Subscription implements SubscriptionLike {
   }
 }
 
+// Even though Subscriptoin only conditionally implements `Symbol.dispose`
+// if it's available, we still need to declare it here so that TypeScript
+// knows that it exists on the prototype when it is available.
+export interface Subscription {
```

Comment:
I have a reasons I put it here actually:

1. Because I'm literally doing this to extend the Subscription interface.
1. I'm planning on a PR to move everything required in `Observable` to a single file (until I can split packages up)

Reply:
I thought we already had `Subscription` interface, but we have `SubscriptionLike` which is not this one.